bin/angstrom/src/components.rs

```diff
@@ -395,13 +394,14 @@ where
     let pool_config = PoolConfig::with_pool_ids(pool_ids);
     let order_storage = Arc::new(OrderStorage::new(&pool_config));
 
-    let _pool_handle = PoolManagerBuilder::new(
+    let _pool_handle = PoolManagerBuilder::new_consensus(
```

Comment:
I've been doing it a little differently in here for example:

https://github.com/SorellaLabs/angstrom/blob/havard/pool-manager/bin/op-angstrom/src/components.rs#L407-L418

Where I type alias the rollup mode and consensus mode and just use regular `new`. Also what I describe in the docs. What do you like more?

Reply:
fixed!